airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -203,6 +211,9 @@ class KubernetesPodOperator(BaseOperator):
         of KubernetesPodOperator.
     :param progress_callback: Callback function for receiving k8s container logs.
         `progress_callback` is deprecated, please use :param `callbacks` instead.
+    :param logging_interval: max time in seconds that task should be in deferred state before
```

Comment:
consider adding the `logging_interval` param to the example DAG?

Reply:
Added example. thank you